custom-recipes/meaningcloud-deep-categorization/recipe.json

```
@@ -1,119 +1,157 @@
+                },
+                {
+                    "value": "Emotion",
+                    "label": "Emotion Recognition"
+                },
+                {
+                    "value": "IntentionAnalysis",
+                    "label": "Intention Analysis"
+                },
+                {
+                    "value": "VoC-Banking",
+                    "label": "Voice of the Customer - Banking"
+                },
+                {
+                    "value": "VoC-Generic",
+                    "label": "Voice of the Customer - Generic"
+                },
+                {
+                    "value": "VoC-Insurance",
+                    "label": "Voice of the Customer - Insurance"
+                },
+                {
+                    "value": "VoC-Retail",
+                    "label": "Voice of the Customer - Retail"
+                },
+                {
+                    "value": "VoC-Telco",
+                    "label": "Voice of the Customer - Telco"
+                },
+                {
+                    "value": "VoE-ExitInterview",
+                    "label": "Voice of the Employee - Exit Interview"
+                },
+                {
+                    "value": "VoE-Organization",
+                    "label": "Voice of the Employee  - Organization"
+                },
+                {
+                    "value": "VoE-Performance",
+                    "label": "Voice of the Employee  - Performance"
+                }
+            ],
+            "defaultValue": "IAB_2.0"
         },
         {
-            "name": "user_model",
-            "label": "User model",
-            "description": "User-defined model to use in the classification.",
-            "type": "STRING",
-            "visibilityCondition" : "model.predefined_model == false"
+            "type": "PRESET",
+            "name": "meaningcloud_connection",
+            "label": "API configuration preset",
+            "parameterSetId": "meaningcloud-connection",
+            "mandatory": true
+        },
+        {
+            "name": "separator_advanced",
+            "label": "Advanced",
+            "type": "SEPARATOR"
+        },
+        {
+            "name": "expert",
+            "label": "Expert mode",
+            "type": "BOOLEAN",
+            "defaultValue": false
         },
         {
             "name": "categories",
             "label": "Number of categories",
-            "description": "Number of categories to obtain.",
+            "description": "Number of categories to obtain",
             "type": "INT",
             "defaultValue": 1,
             "minI": 1,
-            "maxI": 10
+            "maxI": 10,
+            "visibilityCondition": "model.expert == true"
+        },
+        {
+            "name": "user_model",
+            "label": "User model",
+            "description": "User-defined model to use in the classification",
```

```suggestion
            "description": "User-defined model ID to use in the classification",
```
Adding "ID" should clear up for the user the value expected in the field... what do you think?

---

good idea!